pkg/asset/installconfig/platform.go

```diff
@@ -70,15 +71,15 @@ func (a *Platform) queryUserForPlatform() string {
 func (a *Platform) awsPlatform() (*asset.State, error) {
 	return assetStateForStringContents(
 		AWSPlatformType,
-		asset.QueryUser(a.InputReader, "Region:"),
+		asset.QueryUser(a.InputReader, "Region:", "us-east-1"),
 	), nil
 }
 
 func (a *Platform) libvirtPlatform() (*asset.State, error) {
 	return assetStateForStringContents(
 		LibvirtPlatformType,
 		// TODO(yifan): Set the default URI.
```

Comment:
We probably want to drop this TODO.  Regardless of how we address defaults for the libvirt URI here, after this PR lands it should no longer be a TODO.